docs/Support-multiple-reference-genomes.md

```diff
@@ -0,0 +1,35 @@
+# [Introduction](introduction)
+From release 3.1, cBioportal supports multiple reference genomes. This means the hg19 datasets can co-exist with the hg38 ones in the same portal database. We also support mouse genomes (mm10), but it is not recommended to have both human and mouse data in the same instance of the cBioPortal.
```

Comment:
@khzhu I make some change here. I change the version from 2.2 to 3.1. Was 2.2 referred to the db version? I think the portal code version should be used.

I also change the text about mouse genome and recommend not to import human and mouse into the same instance.